scripts/qa_crowbarsetup.sh

```diff
@@ -563,6 +563,12 @@ function get_unclustered_sles12plus_nodes
     echo $sles12plusnodes
 }
 
+function get_non_controller_nodes
+{
+    sles12nodes=get_unclustered_sles12plus_nodes
+    controller_node=get_novacontroller
```

Comment:
What do you mean by "the" controller node?  In HA setup there are multiple controller nodes.